test/doctool/test-doctool-html.js

```diff
@@ -140,21 +139,20 @@ const versions = [
   { num: '0.12.x' },
   { num: '0.10.x' }];
 
-testData.forEach(({ file, html }) => {
+testData.forEach(async ({ file, html }) => {
```

Comment:
We don't need this function returning promises.

```suggestion
testData.forEach(({ file, html }) => {
```